transports/azure-service-bus-netstandard/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Azure Service Bus .Net Standard Transport
+component: ASBS
+tags:
+ - Azure
+reviewed: 2018-06-19
+---
+
+## Azure Service Bus .Net Standard Transport
+
+The Azure Service Bus .Net Standard transport, ASBS in short, is the successor to the [Azure Service Bus transport](transports/azure-service-bus/). It leverages the .Net Standard compatible [Microsoft.Azure.ServiceBus](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/) client SDK.
```

Comment:
> ASBS is the successor to the ASB

and 

> It leverages the .Net Standard compatible client SDK.

are negating each other. It is **not** a successor IMO. It's a completely new and different transport. Just like the new ASB client is not a successor to the old one. I think we better off positioning it as a brand new transport to set expectations right.

Reply:
maybe the term replacement is better?